.github/workflows/node.js.yml

```diff
@@ -0,0 +1,22 @@
+name: Node.js CI
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['18.x', '20.x']
```

Comment:
I'd suggest to update `nvmrc` as well if the support covers `18.x.x`. It is currently set for `20.x.x` only.

Reply:
I'm not sure if we can have multiple versions managed from a single .nvmrc